src/pages/InformationForm.vue

```diff
@@ -98,29 +92,39 @@ export default {
       this.formSchemaData[field]["required"] =
         this.formSchemaData[field].required == "TRUE" ? true : false;
     });
+    this.isUserDataIsComplete();
```

Comment:
isUserDataComplete()

Reply:
ah there is a variable like this. maybe call it checkIfUserDataIsComplete() then?

Reply:
its just a local variable no?